bubuku/health.py

```diff
@@ -3,19 +3,31 @@
 import threading
 from http.server import BaseHTTPRequestHandler, HTTPServer
 
```

Comment:
I think it's not a `health.py` any more. It's more like a http_service or web_controller